sentry-android-core/src/main/java/io/sentry/android/core/internal/util/ScreenshotUtils.java

```diff
@@ -38,13 +43,14 @@ public class ScreenshotUtils {
 
     if (!isActivityValid(activity, buildInfoProvider)
         || activity.getWindow() == null
-        || activity.getWindow().getDecorView() == null
-        || activity.getWindow().getDecorView().getRootView() == null) {
+        || activity.getWindow().peekDecorView() == null
```

Comment:
`m`: I'm wondering if we should rather memoize this into a variable, because the next time we call it, it might've already changed? especially if we are not on the main thread